playbooks/fedora-31/setup-env.yaml

```diff
@@ -16,6 +17,18 @@
           - udisks2
           - podman
 
+    - name: Install bats-support library
+      git:
+      repo: https://github.com/bats-core/bats-support
```

Comment:
Nitpick: shouldn't the `repo`, `depth` and `dest` lines be indented? Because they are arguments for `git`.

Reply:
Yes, this is a mistake I made when squashing commits. Fixed.